src/main/webapp/portal-core/js/portal/layer/querier/QueryTargetHandler.js

```diff
@@ -120,17 +120,53 @@ Ext.define('portal.layer.querier.QueryTargetHandler', {
                 continue;
             }
 
-
             point = Ext.create('portal.map.Point', {
                 latitude : queryTargets[i].get('lat'),
                 longitude : queryTargets[i].get('lng')
             });
 
+
             var shortTitle = cswRecord.get('name');
+            
+            // abbreviate the service provider name
+            var provider = cswRecord.get('contactOrg');   
+            
+            // list of service providers 
+            var SERVICE_PROVIDERS = [
+                'CSIRO',
+                'Geoscience Australia',
+                'Northern Territory',
+                'Queensland',
+                'South Australia',
+                'Tasmania',
+                'Victoria',
+                'Western Australia'
+            ];
+            
+            for (var i = 0; i < SERVICE_PROVIDERS.length; i++) {                
```

Comment:
I started trying to do that and I couldn't figure it out EASILY. I figured that with only 8 String elements a loop isn't going to be a problem.